api/v1alpha1/backendtrafficpolicy_types.go

```diff
@@ -92,6 +93,33 @@ type BackendTrafficPolicySpec struct {
 	// +notImplementedHide
 	// +optional
 	RequestBuffer *RequestBuffer `json:"requestBuffer,omitempty"`
+	// Telemetry configures the telemetry settings for the backend.
+	//
```

Comment:
needs a line here, mentioning that it overrides the setting in the EnvoyProxy resource